docs/releases/changelog.md

```diff
@@ -2940,7 +2940,7 @@ Hover over the info icon to quickly view more information including the commit t
 
 #### Removal of the default repository group `All`
 
-Moderne SaaS no longer provides a default named repository group called `All` that represents all ingested repositories. Customers can [define their own named repository group](../administrator-documentation/moderne-platform/how-to-guides/agent-configuration/configure-organizations-service.md) through Agent to provide this functionality. See the example below.
+Moderne SaaS no longer provides a default named repository group called `All` that represents all ingested repositories. Customers can define their own named repository group through Agent to provide this functionality. See the example below.
```

Comment:
I think this is not true. If users don't provide a `repos.csv` in their agent, then we'll default to `All`. CC @kmccarp to keep me honest.

Reply:
For what it's worth - this was a changelog from awhile ago. So it _may_ have been true when the changelog was released at that time but not _right now_.